packages/vanilla/package.json

```diff
@@ -42,6 +42,9 @@
     "@babel/runtime": "^7.5.5",
     "@carbon/ibmdotcom-styles": "1.1.2",
     "@carbon/ibmdotcom-utilities": "1.2.0-alpha.1070",
+    "@carbon/icon-helpers": "^10.4.0",
+    "@carbon/icons": "^10.6.1",
```

Comment:
for now, these should be `peerDependencies` and not `dependencies`. There is a separate story to address the dependencies and tree shaking issues.